spec/system/admin/reports/orders_and_fulfillment_spec.rb

```diff
@@ -141,7 +142,8 @@
           # 2 rows for order1 + 1 summary row
 
           # setting a time interval to include both orders
-          pick_datetime "#q_completed_at_gt", datetime_start2
+          find("input.datepicker").click
+          select_dates_from_daterangepicker datetime_start2, Time.zone.now
```

Comment:
I am wondering why we don't need to close `flatpickr` here ? Anyway the tests are green, so it's not really important.